src/sage/categories/finite_lattice_posets.py

```diff
@@ -37,6 +41,20 @@ class FiniteLatticePosets(CategoryWithAxiom):
         True
         sage: TestSuite(C).run()
     """
+    @cached_method
+    def extra_super_categories(self):
+        r"""
+        Return a list of the (immediate) super categories of
+        ``self``, as per :meth:`Category.super_categories`.
```

Comment:
```suggestion
        Implement the fact that a finite lattice is a bounded poset.
```
Following https://doc.sagemath.org/html/en/reference/categories/sage/categories/semigroups.html#sage.categories.semigroups.Semigroups.CartesianProducts.extra_super_categories